src/main/java/com/booster/server/CreateVocabularyEntryInput.java

```diff
@@ -0,0 +1,11 @@
+package com.booster.server;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
I think, @Data aggregates more annotations than you currently need. Remove @Data and add only necessary annotations.

Reply:
Done